### PR TITLE
Jelly

### PR DESCRIPTION
**Pull request by bago2k4**
New jelly design and general improvements

Card: https://trello.com/c/tXlzcK0G

### CODE REVIEW

**Comment by bago2k4** [comment]
[Jelly design refresh (new UI)](https://trello.com/c/tXlzcK0G/1073-jelly-design-refresh-new-ui)

**Comment by belucid** [comment]
Reviewing.

**Comment by belucid** [comment]
Logo that's 4897x1105 gets skewed in team settings (worse) and header (better, but still not right).

**Comment by belucid** [comment]
Author attribution is using the first author, it should use the `publisher`.

**Comment by belucid** [comment]
@bago2k4 

- [ ] On a board, sorted by topic, I click a topic pillbox to nav. to the filtered view (which does work) but I get this each time: http://take.ms/0rQ74

**Comment by bago2k4** [comment]
@belucid as stated here https://github.com/facebook/react/issues/6232 that error is caused by an issue in React 15, it's solved in React 16 but since it introduces big differences in errors handling we need to wait for this PR to be merged in Rum before moving on: https://github.com/tonsky/rum/pull/151

Once that's done we need to add a class component (or more) that catches errors thrown by react renders, this way we will be able to provide an alternative UI.

Reference to React 16 changes: https://github.com/facebook/react/releases/tag/v16.0.0 in particular the error handling change: https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html

**Comment by belucid** [comment]
@bago2k4 

On a public board I see the BG of reactions: http://take.ms/lRLiN

**Comment by belucid** [comment]
Reviewing again.

File: site/oc/core.clj [hunk 1]
```diff
@@ -46,7 +46,9 @@
    ;; Static js files
    [:script {:src (pages/cdn "/js/static-js.js")}]
    ;; Bootstrap JavaScript //getf.com/
-   [:script {:src "//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" :type "text/javascript" :integrity "sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" :crossorigin "anonymous"}]])
+   [:script {:src "//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" :type "text/javascript" :integrity "sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" :crossorigin "anonymous"}]
+   ;; jQuery textcomplete needed by Emoji One autocomplete
```
Comment (belucid) [inline site/oc/core.clj]: Comment (jQuery) doesn't match code (JWT)

File: site/oc/pages.clj [hunk 2]
```diff
@@ -424,6 +426,8 @@
           [:div#app [:div.oc-loading.active [:div.oc-loading-inner [:div.oc-loading-heart] [:div.oc-loading-body]]]]
           [:div#oc-error-banner]
           [:div#oc-loading]
+          ;; jQuery textcomplete needed by Emoji One autocomplete
+          [:script {:src "/lib/jwt_decode/jwt-decode.min.js" :type "text/javascript"}]
```
Comment (belucid) [inline site/oc/pages.clj]: Comment (jQuery) doesn't match code (JWT)

File: src/oc/web/actions.cljs [hunk 3]
```diff
@@ -1412,18 +1381,32 @@
...
-      (when-let [ws-link (utils/link-for (:links fixed-activity-data) "interactions")]
-        (ws-ic/reconnect ws-link (jwt/get-key :user-id))))
-    (-> db
-      (dissoc :activity-loading)
-      (assoc-in activity-key fixed-activity-data))))
+  (let [next-db (if (= status 404)
+                  (dissoc db :latest-entry-point)
+                  db)]
+    (when (= status 404)
+      ; (router/redirect-404!)
```
Comment (belucid) [inline src/oc/web/actions.cljs]: Good reason to leave this here commented out?

File: src/oc/web/components/ui/filters_dropdown.cljs [hunk 4]
```diff
@@ -26,32 +26,36 @@
...
+             :on-click #(router/nav! (utils/get-board-url (router/current-org-slug) (:slug board-data)))}
+            (or (:name topic-data) "No topic")]]
+        [:button.mlb-reset.filters-dropdown-button.choice
+          {:type "button"
+           :on-click #(reset! (::show-filters-dropdown s) (not @(::show-filters-dropdown s)))}
+          (cond
+            (or (= board-filters :by-topic)
+                (string? board-filters))
+            [:span "View by " [:span.filter-highlight "Topic"] " "]
+            ; (string? board-filters)
```
Comment (belucid) [inline src/oc/web/components/ui/filters_dropdown.cljs]: Good reason to keep 47-51 commented?

File: src/oc/web/core.cljs [hunk 5]
```diff
@@ -98,12 +97,12 @@
   (pre-routing (:query-params params) true)
   ;; save route
   (router/set-route! ["home"] {:query-params (:query-params params)})
-  (when (jwt/jwt)
-    ;; load data from api
-    (swap! dis/app-state assoc :loading true))
+  ; (when (jwt/jwt)
```
Comment (belucid) [inline src/oc/web/core.cljs]: Need these commented lines?

File: src/oc/web/lib/utils.cljs [hunk 6]
```diff
@@ -665,21 +666,24 @@
...
-  (let [text-string (or text "") ; handle nil
-        unicode-string (.toImage js/emojione text-string)
-        r (js/RegExp "<span " "ig")
-        with-img (.replace unicode-string r "<img ")
-        without-span (.replace with-img (js/RegExp ">(.{1,2})</span>" "ig") (str "alt=\"$1\" />"))]
-
-    (if plain-text
-      without-span
-      #js {"__html" without-span})))
+  ; ;; use an SVG sprite map
```
Comment (belucid) [inline src/oc/web/lib/utils.cljs]: Need the commented sprite map stuff anymore?

**Comment by belucid** [comment]
@bago2k4 looks good. Reactions fix looks good. Just some comments to review about comments. You can merge this once you look at those.

**Comment by belucid** [comment]
@bago2k4 continue to stay on top of Rum / React16 progress. In theory we could get some good performance wins from it.

**Pull request merged**